### PR TITLE
Changed wording to be more clear

### PR DESCRIPTION
**Pull request by MarcelMue**
I tried to make it more clear that port 80 and HTTP only is not related to the example but a general setting.

### CODE REVIEW

File: content/reference/swarm-json.md [hunk 1]
```diff
@@ -220,9 +220,9 @@ As a result, our current component would provide the environment variable `RESTA
 
 ### `domains`
 
-The `domains` key allows you to make components of your application available via HTTP under one or several domain names.
+The `domains` key allows you to make components of your application available via HTTP through port 80 (the HTTP default port) under one or several domain names.
```
Comment (luebken) [inline content/reference/swarm-json.md]: Good addition.

File: content/reference/swarm-json.md [hunk 2]
```diff
@@ -220,9 +220,9 @@ As a result, our current component would provide the environment variable `RESTA
...
 
-The example below makes the given component's exposed port 8080 available under TCP port 80 (the HTTP default port) using the domain name `myexample.gigantic.io`:
+The example below makes the given component's exposed port 8080 available using the domain name `myexample.gigantic.io`:
```
Comment (luebken) [inline content/reference/swarm-json.md]: If it's a common question why not restate it?

**Comment by marians** [comment]
I'd second what @luebken said. Repeating ourselves shouldn't be a problem in this case. Let's rather make sure people see and understand the info.

**Comment by MarcelMue** [comment]
I restated port 80 it is hopefully cleaner now.

**Comment by marians** [comment]
Awesome. Thank you!

**Pull request merged**